lib/internal/loader/ModuleJob.js

```diff
@@ -1,91 +1,93 @@
 'use strict';
 
+const { ModuleWrap } = internalBinding('module_wrap');
```

Comment:
This should be `const { ModuleWrap } = require('internal/process').internalBinding('module_wrap');`